summary.sh

```diff
@@ -4,24 +4,24 @@
 # Use its output to guide your analysis of the results of running ./run-dljc.sh: you must manually
 # examine the results of any project that appears in the "unaccounted for" list.
```

Comment:
This is useful information.  Could you add it to the overview of the experimental methodology that appears, in the README file?  That will help others know how to do experiments.